src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2131,9 +2135,50 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
         _set_remote_or_local_git(cmd, functionapp, resource_group_name, name, deployment_source_url,
                                  deployment_source_branch, deployment_local_git)
 
+    if create_app_insights:
+        try:
+            try_create_application_insights(cmd, functionapp)
+        except Exception:  # pylint: disable=broad-except
+            logger.warning('Error while trying to create and configure an Application Insights for the Function App. '
+                           'Please use the Azure Portal to configure the Application Insights, if needed.')
```

Comment:
Is there no APPInsights CLI command you can reference here as well along with portal?

Reply:
Portal allows to one click, create app insights and integrate with Function Apps. If users manually create it from the CLI, they would need to link them too. So, we defer and recommend Portal directly.

Reply:
Please use the Azure Portal to create and configure ....